mmcv/utils/config.py

```diff
@@ -528,6 +528,15 @@ def __iter__(self):
     def __getstate__(self):
         return (self._cfg_dict, self._filename, self._text)
 
+    def __copy__(self):
+        cls = self.__class__
+        other = cls.__new__(cls)
+
+        for key, value in self.__dict__.items():
+            super(Config, other).__setattr__(key, value)
```

Comment:
Can we directly use `other.__dict__.update(self.__dict__)`?

Reply:
> Can we directly use `other.__dict__.update(self.__dict__)`?

i think so.Let me correct it.